jdaviz/core/helpers.py

```diff
@@ -273,96 +274,96 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
+        try:
+            if loc == "inline":
+                display(self.app)
 
-        scar = Sidecar(**kwargs)
-        with scar:
-            display(self.app)
+            elif loc.startswith('sidecar'):
+                if loc != 'sidecar':
+                    anchor = loc.split(':')[1]
+                    kwargs['anchor'] = anchor
 
-        return scar
-    show_in_sidecar.__doc__ = show_in_sidecar.__doc__.format(
-        sidecar_anchor_values=repr(Sidecar.anchor.values)[1:-1])
+                # If title unset, default to the viz config
+                if 'title' not in kwargs:
+                    kwargs['title'] = self.app.config
 
-    def show_in_new_tab(self, **kwargs):
-        """
-        Display the Jdaviz application in a new tab in JupyterLab.
+                scar = Sidecar(**kwargs)
+                with scar:
+                    display(self.app)
 
-        Additional keywords not listed here are passed into the
-        ``sidecar.Sidecar`` constructor. See
-        `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
-        for the most up-to-date options.
+                return scar
 
-        Parameters
-        ----------
-        title : str, optional
-            The title of the sidecar tab.  Defaults to the name of the
-            application; e.g., "specviz".
+            elif loc == "new browser tab":
+                raise NotImplementedError
 
-        Returns
-        -------
-        sidecar
-            The ``sidecar.Sidecar`` object used to create the tab.
+            elif loc == "popout":
+                raise NotImplementedError
 
-        Notes
-        -----
-        If this method is called in the "classic" Jupyter notebook, the app will
-        appear inline, as only lab has a mechanism to have multiple tabs.
+            else:
+                raise ValueError("Unrecognized display location: " + str(loc))
 
-        See Also
-        --------
-        show_in_sidecar
-        show_inline
+        except Exception as e:
+            warnings.warn(f'Error detected in display: {repr(e)}. Falling back to IPython display',
+                          RuntimeWarning)
+            display(self.app)
+
+    def show_in_sidecar(self, **kwargs):
         """
-        if 'anchor' in kwargs:
-            if 'tab' not in kwargs['anchor']:
-                raise ValueError('show_in_new_tab cannot have a non-tab anchor')
-        else:
-            kwargs['anchor'] = 'tab-after'
-        return self.show_in_sidecar(**kwargs)
+        Preserved for backwards compatibility
+        Shows Jdaviz in a sidecar with the default anchor: right
+        """
+        return self.show(loc="sidecar", **kwargs)
+
+    def show_in_new_tab(self, **kwargs):
+        """
+        Preserved for backwards compatibility
+        Shows Jdaviz in a sidecar in a new tab to the right
+        """
+        return self.show(loc="sidecar:tab-after", **kwargs)
```

Comment:
should we deprecate these somehow so we can remove them eventually?  Were they ever actually used in any documentation/notebooks?

Reply:
Yes, Erik added them into example notebooks but I am not sure who actually uses it.

Reply:
And they've annoyed me by being there ever since! 😆

Reply:
Me too!

Reply:
Seems like there's a desire to remove at least the notebook entries to start deprecating them. I, for one, am happy to oblige!